src/templates/common/googleMapsDirections/index.test.tsx

```diff
@@ -11,10 +11,13 @@ describe('GoogleMapsDirections Component', () => {
 
     render(<GoogleMapsDirections title={title} address={address} />)
 
-    const anchorElement = screen.getByRole('link', {
-      name: /get directions on google maps/i,
-    })
+    // const anchorElement = screen.getByRole('va-link', {
```

Comment:
If tests can't grab and use the web components, that's fine, but we should remove this commented code if we don't intend to use it.

Reply:
I left the old code because I wanted to get feedback on the way it worked previously (commented out section) and the new code. I feel like there may be a better way to do the selector but couldn't figure out how so i gave it a test ID. Test still works as it previously did. Don't know if any of this affects 508 either.